network/wsNetwork.go

```diff
@@ -209,6 +209,7 @@ type OutgoingMessage struct {
 	Action  ForwardingPolicy
 	Tag     Tag
 	Payload []byte
+	Topics  Topics
```

Comment:
why not to use Payload to store serialized topic?

Reply:
The reason for not having the topics paid over the payload is so that the encoding of the topics would be implemented within the network library.